bluesky_queueserver/manager/start_manager.py

```diff
@@ -264,6 +265,19 @@ def start_manager():
         help="Bootstrap server to connect to.",
         default="127.0.0.1:9092",
     )
+
+    parser.add_argument(
+        "--zmq-data-proxy-addr",
+        dest="zmq_data_proxy_addr",
+        type=str,
+        help="The address of ZMQ proxy used to publish data. If the parameter is specified, RE is "
+        "be subscribed to 'bluesky.callbacks.zmq.Publisher' and documents are published via 0MQ proxy. "
```

Comment:
The should say "RE is to be" or "RE is" or "RE will be". Not sure which one you meant.

Reply:
I meant 'is'.

Reply:
Fixed.